coordinator/README.md

```diff
@@ -0,0 +1,12 @@
+# Coordinator service
+
+## Instructions
+
+1. Add `.env` file (see `.env.example`).
+2. Generate RSA key pair with `pnpm run generate-keypair`.
+3. Download zkey files using `pnpm run download-zkeys:{type}` (only test type is available for now).
+4. Make sure you copied RSA public key to your application. This will be needed for encrypting `Authorization` header and coordinator private key for proof generation.
+5. Run `pnpm run start` to run the service.
+6. All API calls must be called with `Authorization` header, where the value is encrypted with RSA public key, you generated before. Header value contains message signature and message digest created by `COORDINATOR_ADDRESS`. Format is `publicEncrypt({signature}:{digest})`.
```

Comment:
nit: maybe remove the comma between `...RSA public Key` and `you generated...`, and add `The` in front of `Format`.